agent/src/index.ts

```diff
@@ -498,6 +505,12 @@ async function startAgent(character: Character, directClient) {
 
         directClient.registerAgent(runtime);
 
+        // Support using API to create NFT
```

Comment:
Will there be a doc that can explain what devs need to do to enable this functionality? If a dev tries to use the NFT generation plugin, will they know they need to uncomment this to have a client to generate NFTs?

Reply:
Yes, there will be documentation explaining how to enable this functionality.

Reply:
> not sure I like it having it's own webserver, seems to duplicate a lot of what direct-client offers in that regard but since direct-client isn't extensible yet, we'll allow it for now. But will likely need to be change this in the future. And convert these into actions agents can take.



After careful consideration, I have decided to take odilitime's [advice](https://github.com/ai16z/eliza/pull/1011#issuecomment-2539510346) to cancel the HTTP service and implement it using AI agent Actions in the future.